Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -916,6 +918,54 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<String>>> InspectorCSSAgent
     return fontFamilyNames;
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>>> InspectorCSSAgent::getCustomFonts()
+{
+    auto customFonts = JSON::ArrayOf<Inspector::Protocol::CSS::FontFace>::create();
+    auto* domAgent = m_instrumentingAgents.persistentDOMAgent();
+
+    for (auto* document : domAgent->documents()) {
+        auto iterator = document->fonts().get().createIterator(document->scriptExecutionContext());
+
+        while (auto font = iterator.next()) {
+
+            String status = ""_s;
+
+            switch (font.get()->status()) {
+            case FontFace::LoadStatus::Unloaded:
+                status = "unloaded"_s;
+                break;
+            case FontFace::LoadStatus::Loading:
+                status = "loading"_s;
+                break;
+            case FontFace::LoadStatus::Error:
+                status = "error"_s;
+                break;
+            case FontFace::LoadStatus::Loaded:
+                status = "loaded"_s;
+                break;
+            default:
+                ASSERT_NOT_REACHED();
+                break;
+            }
+
+            Ref<Inspector::Protocol::CSS::FontFace> result = Inspector::Protocol::CSS::FontFace::create()
+            .setFamily(font.get()->family())
```

Comment:
you can do `font->` instead of calling `.get()`

Style: also please indent all of these
```cpp
auto fontFace = Inspector::Protocol::CSS::FontFace::create()
    .setFamily(font->family())
    .setWeight(font->weight())
    ...
    .release();
customFonts->addItem(WTFMove(fontFace));
```